### PR TITLE
test(NODE-3288): sync command-monitoring spec tests to 3.6

### PR DESCRIPTION
**Pull request by dariakp**
## Description

NODE-3288

See also:
#2837

### CODE REVIEW

File: test/functional/apm.test.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const ignoreNsNotFound = shared.ignoreNsNotFound;
 const loadSpecTests = require('../spec').loadSpecTests;
```
Comment (durran) [inline test/functional/apm.test.js:9]: Maybe we rename this to `command_monitoring.test.js` to match the directory name change? Or would you rather do that as part of the greater test cleanup? I'm fine either way.

**Review by durran — APPROVED** [review]

File: test/functional/apm.test.js (hunk 1 above)
Comment (dariakp) [reply test/functional/apm.test.js:9]: same thoughts here as in my reply on the other PR, but I don't mind if you think it's worth doing now

Comment (durran) [reply test/functional/apm.test.js:9]: Same as well. LGTM.

**Review by nbbeeken — CHANGES_REQUESTED** [review]
LGTM, However some failures on latest are from these spec tests can we skip?

**Comment by emadum** [comment]
> LGTM, However some failures on latest are from these spec tests can we skip?

Some of these are the ones we already skipped in 4.0, there is a commit referenced in NODE-3308 with the tests to skip.

**Review by nbbeeken — APPROVED** [review]

**Pull request merged**